core/client/src/main/java/alluxio/client/file/options/SetAttributeOptions.java

```diff
@@ -31,6 +31,7 @@
   private Boolean mPinned;
   private Long mTtl;
   private TtlAction mTtlAction;
+  // Set attribute for persist will be deprecated in Alluxio 1.4 release.
```

Comment:
add a `LOG.Info` statement to `setPersisted` that prints a deprecation information when invoked the setter is invoked